### PR TITLE
Fix/profile UI bugs

### PR DESCRIPTION
**Pull request by OkoloJohnbosco**
# Fixes Issue/Linear Ticket
https://linear.app/team-grit/issue/FRO2-17/zuri-chat-profile-ui-story-point-3

# Changes proposed

1. Reduce the amount of white space
2. Remove the default background from input field
3. Disabled the jumping effect on user preference section

## What did you do?

1. Reduced the amount of white space on the user preference section.
2. Remove the default background from input field on auto fill
3. Disabled the jumping effect on user preference section

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos
Before
image
image

After
Screenshot 2022-12-01 at 10 14 44

image

### CODE REVIEW

File: packages/main/src/components/protected/topbar/components/TopNavbar.jsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ const TopNavbar = ({ toggleSidebar }) => {
       const res = await authAxios.get(
         `/organizations/${getRealUrl()}/members/${userdef.id}`
       );
-      setUserProfileImage(res.data.data.image_url);
+      setUserProfileImage(res?.data?.data?.image_url);
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/components/TopNavbar.jsx:154]: what happens if there is no profile image?
set it to a dummy profile

File: packages/main/src/components/protected/topbar/styles/NotificationPreference.module.css [hunk 2]
```diff
@@ -126,6 +126,12 @@
 .allPreferences {
   width: 100%;
 }
+.buttonToggle {
+  /* background-color: red; */
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/styles/NotificationPreference.module.css:130]: remove commented styles

File: packages/main/src/components/protected/topbar/styles/NotificationPreference.module.css [hunk 3]
```diff
@@ -126,6 +126,12 @@
...
+
+  height: 90vh;
+  /* padding-top: 20px; */
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/styles/NotificationPreference.module.css:133]: commented styles

File: packages/main/src/components/protected/topbar/styles/ProfileModal.module.css [hunk 4]
```diff
@@ -224,25 +224,27 @@
...
+.modalFull,
+.modalFullScroll {
   width: 80%;
   max-width: 885px;
   background: #fff;
   border-radius: 4px;
   overflow: hidden;
-  padding-bottom: 20px;
-  height: 90vh;
+  /* height: 90vh; */
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/styles/ProfileModal.module.css:234]: commented styles

File: packages/main/src/components/protected/topbar/styles/ProfileModal.module.css [hunk 5]
```diff
@@ -224,25 +224,27 @@
...
+}
+.modalFullScroll {
+  overflow: auto !important;
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/styles/ProfileModal.module.css:237]: important?

File: packages/main/src/components/protected/topbar/styles/ProfileModal.module.css [hunk 6]
```diff
@@ -224,25 +224,27 @@
...
   .modalContent {
-    width: 65%;
+    width: 70%;
   }
 }
 
 .modalCont {
   overflow-y: auto;
-  height: 100%;
+  /* height: 100%; */
```
Comment (ickynavigator) [inline packages/main/src/components/protected/topbar/styles/ProfileModal.module.css:247]: commented styles

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Comment by billmal071** [comment]
Looks good.

**Review by ickynavigator — APPROVED** [review]

**Pull request merged**